show-land/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anvogue</title>
+
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+
+   <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+    <div class="background-color">
+
+        <div class="container">
+            <header class="header ">
+                <h1 class="logo">Anvogue</h1>
+                <nav class="main-nav">
+                    <ul class="main-nav__list">
+                        <li> <a class="main-nav__link" href="#">demos</a> </li>
+            <li><a class="main-nav__link" href="#">mobile view</a> </li>
+            <li><a class="main-nav__link" href="#">features</a> </li>
+            <li><a class="main-nav__link" href="#">help & support</a> </li>
+            <li><a class="main-nav__link btn btn-buttom nav-cta" href="#">purchase now</a> </li>
+        </ul>
+    </nav>
+</header>          
+</header>
```

Comment:
1. If you want to create a header you must use this code

`<header></header>`
or
`<div class="header"></div>`

2. Write your code more neatly
You can use **prettier** in vscode to tidy up your code

3. use capitalcase
For example: btnButtom